src/main/java/io/github/thebusybiscuit/slimefun4/integrations/McMMOIntegration.java

```diff
@@ -54,6 +55,19 @@ public void onItemSalvage(McMMOPlayerSalvageCheckEvent e) {
         }
     }
 
+    @EventHandler(ignoreCancelled = true)
+    public void onItemRepair(McMMOPlayerRepairCheckEvent e) {
+        ItemStack repaired = e.getRepairedObject();
+        ItemStack material = e.getRepairMaterial();
+
+        if (isSalvageable(repaired) && isSalvageable(material)) {
+            return;
+        }
+
+        e.setCancelled(true);
+        Slimefun.getLocalization().sendMessage(e.getPlayer(), "anvil.mcmmo-repair");
```

Comment:
This message does not exist in the `messages.yml`, does it?

Reply:
It doesn't exist yet